drivers/shtc1/shtc1.c

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright 2017, RWTH-Aachen, Steffen Robertz, Josua Arndt
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+#include "shtc1.h"
+
+#define ENABLE_DEBUG	(0)
+#include "debug.h"
+
+
+/*
+ * does a crc check and returns 0 for passed and -1 for failed
+ */
+int8_t _check_crc(uint8_t *rec_values, uint8_t right_crc)
+{
+	uint8_t crc = 0xFF;
+	uint8_t current_byte;
+	uint8_t bit;
+
+	for (current_byte = 0; current_byte < 2; current_byte++)
+	{
+		crc ^= (rec_values[current_byte]);
+		for (bit = 8; bit > 0; bit--)
+		{
+			if (crc & 0x80)
```

Comment:
Please add curly braces to the it/else statement. Maybe have another look at our [coding conventions](https://github.com/RIOT-OS/RIOT/wiki/Coding-conventions).